distribution/test/itests/test-itests-alliance/src/test/java/org/codice/alliance/test/itests/NsiliSourceTest.java

```diff
@@ -68,14 +67,6 @@
 
   private Thread mockServerThread;
 
-  @Override
-  protected Option[] configureCustom() {
```

Comment:
This isn't needed as this bundle is started by the test dependencies feature file